File: 백준/Silver/2563. 색종이/색종이.py
arr = [[0] * 100 for i in range(100)]

n = int(input())
while(n > 0):
    a, b = map(int, input().split())
    for i in range(a - 1, a + 9):
        for j in range(b - 1, b + 9):
            arr[i][j] = 1
    n -= 1
    
cnt = 0
for i in range(100):
    for j in range(100):
        if(arr[i][j] == 1):
            cnt += 1
print(cnt)

File: 백준/Silver/2563. 색종이/색종이.c
#include <stdio.h>

int main(void) {
	int num, arr[100][100] = { 0, };
	scanf("%d", &num);
	int x, y;
	while (num > 0) {
		scanf("%d %d", &x, &y);
		for (int i = x - 1; i < x + 9; i++) {
			for (int j = y - 1; j < y + 9; j++) {
				arr[i][j] = 1;
			}
		}
		num--;
	}
	int cnt = 0;
	for (int i = 0; i < 100; i++) {
		for (int j = 0; j < 100; j++) {
			if (arr[i][j] == 1) cnt++;
		}
	}
	printf("%d", cnt);
							 
	return 0;				 
}	